doc/cascadia/profiles.schema.json

```diff
@@ -0,0 +1,449 @@
+{
+  "$id": "https://github.com/microsoft/terminal/terminal.profile.schema.json",
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "title": "Microsoft Terminal Profile Settings",
```

Comment:
Shouldn't this be "Microsoft's Windows Terminal Settings Profile Schema" or similar?